### PR TITLE
Add header tooltips

### LINKED ISSUE #4513: Add tooltips R&R form columns
It should be clear what each column in the R&R form means, explanatory tooltips would be helpful, especially for:

- Columns populated by the system (how did it get this number?)
- Calculated columns (what is the calculation)
- Expiry - this is the expiry date of the oldest batch of that item in stock

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes #4513

# 👩🏻‍💻 What does this PR do?

Adds some tooltipping to the headers in the R&R detail table:

image

## 💌 Any notes for the reviewer?

# 🧪 Testing

- [ ] Check that tooltips show
- [ ] review text of them

# 📃 Documentation

- [x] **Part of an epic**: documentation will be completed for the feature as a whole
- [ ] **No documentation required**: no user facing changes or a bug fix which isn't a change in behaviour
- [ ] **These areas should be updated or checked**: 
  1.
  2.

### CODE REVIEW

File: client/packages/common/src/intl/locales/en/programs.json [hunk 1]
```diff
@@ -11,11 +11,29 @@
   "control.search.no-results-label": "No results",
   "control.search.reset-button": "Unlink matched patient",
   "control.search.searching-label": "Searching...",
+  "description.expiry": "The closest expiry date of current stock on hand",
+  "description.rnr-consumed-adjusted": "Shows the consumption adjusted to account for stock out days. =(Consumed x 30)/(30 - Stock out duration)",
+  "description.rnr-amc": "The Average Monthly Consumption is calculated using the Adjusted consumption value, using this R&R form and the previous two, if available",
+  "description.rnr-initial-balance": "The total stock on hand for this item at the beginning of the period. This is taken from the previous R&R form, if available, or calculated using the stock on hand at the period start date.",
+  "description.rnr-final-balance": "Final balance of the stock at the end of the period. =Initial balance + Received - Consumed ± Adjustments",
+  "description.rnr-maximum-quantity": "The target quantity of stock to have on hand. =AMC x 2",
+  "description.rnr-requested-quantity": "The quantity to be ordered. The calculated value is =Maximum quantity - Final balance",
```
Comment (lache-melvin) [inline client/packages/common/src/intl/locales/en/programs.json:20]:
```suggestion
  "description.rnr-requested-quantity": "The quantity to be ordered. =Maximum quantity - Final balance",
```
Just to be consistent with other fomulae

File: client/packages/common/src/intl/locales/en/programs.json [hunk 2]
```diff
@@ -11,11 +11,29 @@
   "control.search.no-results-label": "No results",
   "control.search.reset-button": "Unlink matched patient",
   "control.search.searching-label": "Searching...",
+  "description.expiry": "The closest expiry date of current stock on hand",
+  "description.rnr-consumed-adjusted": "Shows the consumption adjusted to account for stock out days. =(Consumed x 30)/(30 - Stock out duration)",
+  "description.rnr-amc": "The Average Monthly Consumption is calculated using the Adjusted consumption value, using this R&R form and the previous two, if available",
+  "description.rnr-initial-balance": "The total stock on hand for this item at the beginning of the period. This is taken from the previous R&R form, if available, or calculated using the stock on hand at the period start date.",
+  "description.rnr-final-balance": "Final balance of the stock at the end of the period. =Initial balance + Received - Consumed ± Adjustments",
```
Comment (lache-melvin) [inline client/packages/common/src/intl/locales/en/programs.json:18]:
```suggestion
  "description.rnr-final-balance": "Stock at the end of the period. =Initial balance + Received - Consumed ± Adjustments",
```

File: client/packages/common/src/intl/locales/en/programs.json [hunk 3]
```diff
@@ -11,11 +11,29 @@
   "control.search.no-results-label": "No results",
   "control.search.reset-button": "Unlink matched patient",
   "control.search.searching-label": "Searching...",
+  "description.expiry": "The closest expiry date of current stock on hand",
+  "description.rnr-consumed-adjusted": "Shows the consumption adjusted to account for stock out days. =(Consumed x 30)/(30 - Stock out duration)",
+  "description.rnr-amc": "The Average Monthly Consumption is calculated using the Adjusted consumption value, using this R&R form and the previous two, if available",
+  "description.rnr-initial-balance": "The total stock on hand for this item at the beginning of the period. This is taken from the previous R&R form, if available, or calculated using the stock on hand at the period start date.",
```
Comment (lache-melvin) [inline client/packages/common/src/intl/locales/en/programs.json:17]:
```suggestion
  "description.rnr-initial-balance": "Stock on hand at the beginning of the period. This is taken from the previous R&R form, if available, or calculated using the stock on hand at the period start date.",
```

File: client/packages/common/src/intl/locales/en/programs.json [hunk 4]
```diff
@@ -11,11 +11,29 @@
...
+  "description.rnr-final-balance": "Final balance of the stock at the end of the period. =Initial balance + Received - Consumed ± Adjustments",
+  "description.rnr-maximum-quantity": "The target quantity of stock to have on hand. =AMC x 2",
+  "description.rnr-requested-quantity": "The quantity to be ordered. The calculated value is =Maximum quantity - Final balance",
+  "description.rnr-consumed": "The quantity of this item which is part of an outbound shipment or prescription which was sent during the requisition period.",
```
Comment (lache-melvin) [inline client/packages/common/src/intl/locales/en/programs.json:21]:
```suggestion
  "description.rnr-consumed": "The quantity of this item distributed during the period (via Outbound Shipments or Prescriptions)",
```

File: client/packages/common/src/intl/locales/en/programs.json [hunk 5]
```diff
@@ -11,11 +11,29 @@
...
+  "description.rnr-received": "The quantity of this item which is part of an inbound shipment which was received during the requisition period.",
```
Comment (lache-melvin) [inline client/packages/common/src/intl/locales/en/programs.json:22]:
```suggestion
  "description.rnr-received": "The quantity of this item received during the period (via Inbound Shipments)",
```

**Review by lache-melvin — APPROVED** [review]
Looks rad to me thanks Mark! Had a play with some wording but also looks good as is!

**Comment by mark-prins** [comment]
Awesome! Thanks @lache-melvin, have committed with a minor tweak 🙏

**Pull request merged**